src/serve_rendered.js

```diff
@@ -1020,8 +1020,23 @@ export const serve_rendered = {
                 const format = extensionToFormat[extension] || '';
                 createEmptyResponse(format, '', callback);
               }
+            } else if (protocol === 'file') {
```

Comment:
change 'file' to 'files' for consitency. this will require you use files:// in your style

Reply:
I'm not sure about "files://" vs. "file://". The later is a commonly valid URI protocol (like "http://"), the other one a real custom protocol.